apps/src/templates/instructions/RubricField.jsx

```diff
@@ -0,0 +1,92 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import color from '@cdo/apps/util/color';
+import Radium from 'radium';
+
+const styles = {
+  detailsArea: {
+    width: '100%'
+  },
+  rubricHeader: {
+    fontSize: 13,
+    marginLeft: 10,
+    color: color.black,
+    fontFamily: '"Gotham 5r", sans-serif'
+  },
+  performanceLevelHeader: {
+    display: 'flex',
+    justifyContent: 'flex-start',
+    flexDirection: 'row',
+    padding: '4px 10px',
+    ':hover': {
+      border: 'solid 1px' + color.light_cyan,
```

Comment:
you can use string interpolation here instead of concatenation:

```
border: `solid 1px ${color.light_cyan}`
```

(same on line 34)